app/admin/license_permission.rb

```diff
@@ -0,0 +1,3 @@
+ActiveAdmin.register LicensePermission do
+  permit_params :name, :description, :icon
+end
```

Comment:
Was this all that was needed to add CRUD operations for the LicensePermission model? :-)